cvat-core/tests/mocks/dummy-data.mock.js

```diff
@@ -2662,8 +2681,9 @@ const frameMetaDummyData = {
         size: 5002,
         image_quality: 50,
         start_frame: 0,
-        stop_frame: 5001,
+        stop_frame: 499,
```

Comment:
Why did you change this value?

Reply:
Couse `frameDataCache` stores job-related data meta now, so it's job-related data start/stop frames.